elements/pfe-card/src/pfe-card.scss

```diff
@@ -100,6 +114,21 @@ $variables: (
 
 // Targets the wrappers in the shadow DOM
 .pfe-card {
+  &--overlay {
+      background-color: #{pfe-local($cssvar: BackgroundColor, $region: overlay)};
+      position: absolute;
```

Comment:
@starryeyez024 @chrisdo1 Do you think this set of styles would make for a good placeholder?  Do these styles have more uses than just as an overlay?

```
%overlay {
      position: absolute;
      top: 0;
      right: 0;
      bottom: 0;
      width: 100%;
      left: 0;
      height: 100%;
}
```

Reply:
@starryeyez024 bump